README.md

```diff
@@ -26,16 +26,38 @@ typo3cms phinx:seed:run
 
 Notice that these wrapper commands are executed by TYPO3, thus the full API like `DataHandler` can be used in migrations.
 
+## Migrations
+
 The following paths are used for migrations:
 
 * `typo3conf/ext/*/Migrations/Phinx`
 * `typo3conf/ext/*/Classes/Migrations/Phinx`
 
+Examples to create a migration in a TYPO3 project:
+
+* `vendor/bin/typo3cms phinx:create --path packages/provider/Migrations/Phinx MyMigration`
+* `vendor/bin/typo3cms phinx:create --path packages/provider/Classes/Migrations/Phinx MyMigration`
+
+**Note**
+
+If a `Migrations` directory exists already, you can omit `--path`.
```

Comment:
If you have other extensions (own or 3rd party) which provide Phinx migrations, this is not true however. Phinx will notice that there's more than one path and prompt for the desired location. So if in doubt, one should always use the `--path` option.

Reply:
I have rephrased it. Is it better now? If not, then I will give it some further thoughts asap. This change is a shotgun, hoping it will be better - did not gave it a lot of thoughts because I need to do some other stuff.